docs/code-samples/getting-started/daml/Workflow/DvP.daml

```diff
@@ -51,7 +51,8 @@ template Proposal
         -- INSTRUCT_BEGIN
         (containerCid, [recInstructionCid, payInstructionCid]) <-
           exercise settlementFactoryCid Settlement.Instruct with
-            instructors = fromList [proposer, counterparty]
+            instructor = proposer
+            consenters = fromList [counterparty]
```

Comment:
`singleton` might be slightly more efficient, but i'm not 100% sure about that

Reply:
I changed that (and in multiple other places as well)